### PR TITLE
Add text-based filter in members page for Member names

### LINKED ISSUE #1239: "Member name" filter in Members list
### Describe the problem

Currently, the only way to look for a member (Members list) is via search input. That does not provide enough flexibility if we think of use cases like finding duplicates/similar member names, or making use of filter operators (contains, does not contain, etc.)

### Describe the improvement

Add a new text-based filter - Member name - to Members list. Position in filters list: 1st
Screenshot 2023-08-08 at 14 04 48

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by peoray**
# Changes proposed ✍️

### What
Fixes #1239 

### <samp>🤖 Generated by Copilot at 1cfca44</samp>

Added a new `memberName` filter for members in the frontend UI. Created a new file `memberName/config.ts` to define the filter config and imported it in `filters/main.ts`.
​

### <samp>🤖 Generated by Copilot at 1cfca44</samp>

> _`memberName` filter_
> _config file defines options_
> _autumn leaves sorting_

### <samp>🤖 Generated by Copilot at 1cfca44</samp>

*  Add `memberName` filter config for filtering members by name ([link](https://github.com/CrowdDotDev/crowd.dev/pull/1427/files?diff=unified&w=0#diff-ad93c3671a203d1c70a3ef431a8caf74c92994f8b8acd40f4950efd338ab22c9R1-R31))
*  Import and use `memberName` filter config in `main.ts` ([link](https://github.com/CrowdDotDev/crowd.dev/pull/1427/files?diff=unified&w=0#diff-9764bba874a0ee03488cf483fab4b7144d11c402793e69730e1a5508f1451535L15-R18))

## Checklist ✅
- [x] Label appropriately with `Feature`, `Improvement`, or `Bug`.
- [ ] Add screehshots to the PR description for relevant FE changes
- [ ] New backend functionality has been unit-tested.
- [ ] API documentation has been updated (if necessary) (see [docs on API documentation](https://docs.crowd.dev/docs/updating-api-documentation)).
- [ ] [Quality standards](https://github.com/CrowdDotDev/crowd-github-test-public/blob/main/CONTRIBUTING.md#quality-standards) are met.

### CODE REVIEW

**Comment by peoray** [comment]
@joanagmaia this is good to go, hopefully

**Review by joanagmaia — CHANGES_REQUESTED** [review]

File: frontend/src/modules/member/config/filters/main.ts [hunk 1]
```diff
@@ -12,8 +12,10 @@ import joinedDate from './joinedDate/config';
 import lastActivityDate from './lastActivityDate/config';
 import reach from './reach/config';
 import tags from './tags/config';
+import memberName from './memberName/config';
 
 export const memberFilters: Record<string, FilterConfig> = {
+  memberName,
```
[the remark this answers is not in the document]
Comment (peoray) [reply frontend/src/modules/member/config/filters/main.ts:18]: the issue stated it should be first on the list. did you see that? just confirming before going ahead

[the remark this answers is not in the document]
Comment (joanagmaia) [reply frontend/src/modules/member/config/filters/main.ts:18]: Yeah I missed it, apologies for that. It's ok as it is 😄

**Review by joanagmaia — APPROVED** [review]
LGTM 🚀 
Will try to ship it today 😄

**Pull request merged**